std_atmosphere.m

```diff
@@ -16,38 +16,42 @@
 % t - kelvin
 
 
-%  [ rho, p,nu, a, t] = std_atmosphere( altitude_m,model_num)
-% uni: 0 is the meter, 1 = ft 
-
 
 GAMMA = 1.4; % or replace with handle if we are not using constant
 R_AIR = 286.9; %j/kg.K
+mu_0_earth = 0.000017332654; %Pa.s
+earth_t0 = 288.11; %K
+earth_ts = 110.33; %K
 
 p = 0;
 t = 0;
-
+mu = 0;
 
 if altitude < 86000 %Troposphere
     [rho,a,t,p,nu,ZorH]=stdatmo(altitude,0,'si');
     return
 % do some stuff here for altitude abvoe 86km
 elseif altitude < 91000 
+    altitude = altitude/1000; %Equations are for z in km
     t = 186.8673;
-    p = exp(2.156582e-06*altitude^3 -4.836957e-04*altitude^2-0.2609971*altitude + 13.47530);
+    p = exp(2.156582e-06*altitude^3 -4.836957e-04*altitude^2-0.1425192*altitude + 13.47530);
     rho = exp(-3.322622e-06*altitude^3 +9.111460e-04*altitude^2-0.1425192*altitude + 5.944694);
+    mu = mu_0_earth*((t/earth_t0)^1.5)*((earth_t0+earth_ts)/(t+earth_ts));
 elseif altitude < 100001
+    altitude = altitude/1000; %Equations are for z in km
     delta_z = altitude - 91000;
     little_a = -19942.9;
-    t = 263.1905-76*sqrt(1-((delta_z/little_a)^2));
+    t = 263.1905-76.3232*sqrt(1-((delta_z/little_a)^2));

```

Comment:
delta_z might be negative here... since you did altitude /1000 already, i thought the z here had to be meters. RIP us.